packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -123,18 +259,28 @@ exit 0
 %{foreman_apipie_cache_foreman}
 %{foreman_apipie_cache_plugin}
 %{foreman_dir}/script/foreman-debug.d/60-dynflow_debug
-%config(noreplace) %{_root_sysconfdir}/logrotate.d/%{gem_name}
-%doc %{gem_instdir}/LICENSE
-
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{gem_name}
```

Comment:
This should also be in `%{_root_sysconfdir}` so it's in `/etc/logrotate.d` rather than `/opt/..../etc/logrotate.d`